### ISSUE TITLE
Point to `workflow.pacta` as a base image

### ISSUE THREAD

**Issue by jdhoffa** [RMI-PACTA/workflow.prepare.pacta.indices#41]
A new repository called `workflow.pacta` seems to (maybe?) have the goal of superseding the existing `workflow.transition.monitor` workflow. Whatever the "core PACTA engine" ends up being, this repository should make use of that as a base image. 

[AB#10658](https://dev.azure.com/RMI-PACTA/d77fc8f8-abac-42ae-9e31-846f9901290d/_workitems/edit/10658)

**Comment by AlexAxthelm** [comment]
makes sense to me. Maybe wait a day or two though, since I'm planning on doing some work on workflow.pacta later today and tomorrow

**Comment by jdhoffa** [comment]
All good. The main question was more or less is `workflow.pacta` an experiment, or is it superseding existing PACTA. Sounds like the latter is true.

**Comment by cjyetman** [comment]
> All good. The main question was more or less is `workflow.pacta` an experiment, or is it superseding existing PACTA. Sounds like the latter is true.

I think we need to think about this carefully. Although long term I hope workflow.pacta supersedes existing PACTA workflows, in the medium term we still need to support workflow.transition.monitor. And my super cautious side thinks we should build indices for/on the same workflow they are intended to be used with.

**Comment by jdhoffa** [comment]
This is the discussion I wanted to spark! haha

**Comment by jdhoffa** [comment]
Important takeaway from a recent discussion:
DO NOT DO THIS UNTIL PACTA COP IS OVER.

**Comment by AlexAxthelm** [comment]
> DO NOT DO THIS UNTIL PACTA COP IS OVER.

DO NOT DO THIS UNTIL _DATA PREP_ for PACTA COP IS OVER.

**Comment by jdhoffa** [comment]
Given that in the past we have had to run data prep upwards of 20 times, I don't know if there is a clear point that data prep is "over"

So I think best to wait until COP is actually over haha
